controllers/user.controller.js

```diff
@@ -34,8 +38,24 @@ const updateUser = async (req, res, next) => {
     }
 };
 
+const uploadAvatar = async (req, res, next) => {
+    try {
+        const user = req.query.user_id;
+        await service.updateUser({id: user, avatar: `${user}.jpg`});
+        const data = req.body.image.replace("data:image/png;base64,", "");
+        await fs.writeFileSync(path.resolve(__dirname, "../uploads/avatars", `${user}.jpg`), data, "base64");
```

Comment:
@ikadushkina use async fs analogue for writing files, as the sync operation will block the event loop. Btw, currently the operation is sync, so the await is useless here.